### DISCUSSION TITLE
Cannot combine RefinableString search term with Path

### DISCUSSION THREAD

**Discussion by Emmanueladegor** [microsoft-search/pnp-modern-search#3360] [Q&A] [answered]
Hello wonderful team, 

CC: @wobba 

My issue:

When I use ({searchTerm} path: "document library url") for my Query template, it returns results. However, I want to limit the search to a column. But when I modify it to 
RefinableString12:({searchTerm} path: "document library URL"), it does not return any result. 

Question: How can I make: RefinableString12:({searchTerm} path: "document library URL") return results, when documents are searched?

Thank you

**Comment by kasperbolarsen** [comment]
Hi, as usual the answer is "it depends". 
What is the RefinableString mapped to?

In general I recommend using one if the tools here when debugging a search query: https://microsoft-search.github.io/pnp-modern-search/using-query-tools-to-verify-issues/

**Reply by Emmanueladegor** [reply]
Hello @kasperbolarsen ,

Thanks for the helpful reply. If I remove “Path”, the refinableString search works. However it searches all libraries on the site. The challenge is, it seems combining Path and refinableString isn’t achievable.

So,

- Without RefinableString, the searchTerm works, it searches the document library, based on the path I specify.
- Without specifying a path, the refinableString works, it searches the mapped column on any library if finds.
- But then, I want to search a certain column on a specific path, that’s where I am challenged.

Does this provide clarity?

Thank you

**Reply by kasperbolarsen** [reply, marked as answer]
Is RefinableString12:({searchTerm} path: "document library URL") a typo?

I guess you want RefinableString12:{searchTerm} AND path:"document library URL"
?

**Reply by Emmanueladegor** [reply]
That did the magic @kasperbolarsen , 

Yes, it was an intentional typo. I didn't know how to merge those two.

Appreciate you, 
Thanks

**Reply by wobba** [reply]
I think you want to keep () in case of multi term input. `RefinableString12:({searchTerms}) AND path:"document library URL"` The important part is not to have any spaces in the property query before or after the `:` or `=` etc. And have the `s`at the end of searchTerms :)

**Reply by Emmanueladegor** [reply]
@wobba - Appreciate you,  Thanks!
